### PR TITLE
Rename the auto-confirm flag to yes

### PR DESCRIPTION
**Pull request by billyb2**
### Change Summary

What and Why:

This makes auto-confirming much more consistent across the CLI. The tests were purposely not changed to test backwards compatibility.

How:

Use `flag.Yes()` everywhere, and make `--auto-confirm` and alias for `--yes` so it shouldn't break anything

Related to:

---

### Documentation

- [ ] Fresh Produce
- [ ] In superfly/docs, or asked for help from docs team
- [x] n/a

### CODE REVIEW

**Comment by michaeldwan** [comment]
The change looks good, but I'd suggest using `--force` rather than `--yes` since it's usually the flag to do something potentially destructive without confirmation. `--yes` can be an alias. What do you think?

File: internal/flag/flag.go [hunk 1]
```diff
@@ -327,9 +332,15 @@ func Yes() Bool {
 		Name:        flagnames.Yes,
 		Shorthand:   "y",
 		Description: "Accept all confirmations",
+		Aliases:     []string{"auto-confirm"},
 	}
 }
 
+// Checks whether the --yes flag or the --auto-confirm flag is present
+func HasYes(ctx context.Context) bool {
```
Comment (dangra) [inline internal/flag/flag.go:340]: The convention we have been following so far is to call the function `flag.GetFlagName()` for `flag.FlagName()`, in this case it should be `flag.GetYes()`

**Comment by billyb2** [comment]
> The change looks good, but I'd suggest using `--force` rather than `--yes` since it's usually the flag to do something potentially destructive without confirmation. `--yes` can be an alias. What do you think?

Hm, I personally prefer `--yes` since it's already the convention in flyctl (as well as many other CLIs), though I see the appeal of `--force`. If you think force would be better though, I'll change it to that.

**Comment by michaeldwan** [comment]
> Hm, I personally prefer `--yes` since it's already the convention in flyctl (as well as many other CLIs), though I see the appeal of `--force`. If you think force would be better though, I'll change it to that.

Okay I've changed my mind... `--yes` makes sense it's only for skipping confirmation prompts, which appears to be how we're using `--yes` and `--force` right now. So sticking with yes makes sense. Carry on!

File: internal/command/deploy/deploy.go [hunk 2]
```diff
@@ -60,10 +60,7 @@ var CommonFlags = flag.Set{
 		Shorthand:   "e",
 		Description: "Set of environment variables in the form of NAME=VALUE pairs. Can be specified multiple times.",
 	},
-	flag.Bool{
-		Name:        "auto-confirm",
-		Description: "Will automatically confirm changes when running non-interactively.",
-	},
+	flag.Yes().SetDescription("Will automatically confirm changes when running non-interactively."),
```
Comment (michaeldwan) [inline internal/command/deploy/deploy.go:63]: Is there a benefit to changing the flag description for specific commands, or can we use the same flag description for everything? eg "Skip confirmation prompt"

Comment (billyb2) [reply internal/command/deploy/deploy.go:63]: No not necessarily, I mostly just changed it so this didn't change much, but honestly it doesn't matter much.

**Review by michaeldwan — APPROVED** [review]
This is excellent. Do you think you can come back and update the commands that currently have a `--force` flag (if it makes sense to do so)?

**Pull request merged**